Neos.Neos/Classes/Service/NodeTypeSchemaBuilder.php

```diff
@@ -54,19 +53,24 @@ public function generateNodeTypeSchema()
             'constraints' => $this->generateConstraints()
         ];
 
+        // We need to include abstract nodetypes as the UI required some basic types like Neos.Neos:Document and Neos.Neos:Content
         $nodeTypes = $this->nodeTypeManager->getNodeTypes(true);
```

Comment:
what if we add these required abstract nodetypes manually to the list and otherwise only use the non abstract ones?

Reply:
You mean the list in `Neos.Neos.Ui.nodeTypeRoles` ;) 

Yes that would be better, sadly wrong package, meaning we would have to duplicate the setting.

Reply:
id argue this service is in the wrong package.

hmm eventually we will move this over to the ui - but as this is supposed to be just a bugfix we cant do it now. but we could hackily use the settings already even if this speaks against everything ^^

Reply:
OK, I made the change and the Neos.UI now works generally without abstract nodetypes except the ones from its settings which I include separately.